src/index.html

```diff
@@ -1,12 +1,382 @@
 <!doctype html>
-<html lang="en">
+<html lang="en"
+      class="page">
+
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="
+      UTF-8">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet"
+          href="./styles/main.scss">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <a class="home"
+         href="#home"
+         alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#APPLE" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#SAMSUNG" class="nav__link margin-left">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#SMARTPHONES"
+               class="nav__link margin-left">smartphones</a>
```

Comment:
Fix code style, check yourself with checklist guide 
Fix all cases
```suggestion
            <a 
              href="#SMARTPHONES"
              class="nav__link margin-left"
            >
              smartphones
            </a>
```